src/scss/sections/_testimonials.scss

```diff
@@ -0,0 +1,5 @@
+.list-logo {
```

Comment:
please name the class as the section, testimonials is enough using as class for the main SECTION